pkg/notaryv2/notaryv2.go

```diff
@@ -128,5 +136,118 @@ func (v *notaryV2Verifier) verifyOutcomes(outcomes []*notation.VerificationOutco
 }
 
 func (v *notaryV2Verifier) FetchAttestations(ctx context.Context, opts images.Options) (*images.Response, error) {
-	return nil, errors.Errorf("not implemented")
+	v.log.V(2).Info("fetching attestations", "reference", opts.ImageRef)
```

Comment:
Would be good to break this method into smaller methods:
- fetchAttestation
- verifyAttestators
- extractStatements